src/main.rs

```diff
@@ -148,14 +148,22 @@ fn main() -> Result<()> {
                 }
                 Err(error) => {
                     error!("{error}");
+                    error
```

Comment:
Ah! Thanks for the tip. I haven't used anyhow before, so I wasn't sure how to retrieve the rest of the context from the error. That's nice.